src/chrome/content/rules/administrator.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="administrator.de">
+<target host="*.administrator.de" />
+<rule from="^http:" to="https:" />
+<test url="http://www.administrator.de/" />
```

Comment:
But if someone goes to http://www.administrator.de he isnt redirectet, or i am wrong?